Functions/Administration/xASL_adm_SortDicomToFolders.m

```diff
@@ -47,50 +47,53 @@ function xASL_adm_SortDicomToFolders(pathDICOM, nDirLayers, bUseDCMTK, bVerbose)
 	end
 else
 	% Gather a list of all files in the directory
-	Flist   = xASL_adm_GetFileList(pathDICOM,'^.*.(?!(xlsx|ini|json))$','FPListRec',[0 Inf]);
+	Flist   = xASL_adm_GetFileList(pathDICOM,'.*','FPListRec',[0 Inf]);
```

Comment:
No, you want to get all files but not xls, ini, or json, so this change should be reverted